ui/admin/app/components/form/target/details/index.js

```diff
@@ -29,4 +30,13 @@ export default class FormTargetComponent extends Component {
   get icon() {
     return icons[this.args.model.type];
   }
+
+  /**
+   * determines when the worker filter deprecation message should be shown
+   * @type {boolean}
+   */
+  @computed('args.model.{isNew,worker_filter}')
```

Comment:
Is this `computed` needed?

Reply:
Nope. I will remember that one day. Removing.